src/main/java/org/apache/commons/collections4/ListUtils.java

```diff
@@ -744,6 +745,23 @@ public static String longestCommonSubsequence(final CharSequence charSequenceA,
         return UnmodifiableList.unmodifiableList(list);
     }
 
+    /**
+     * Returns a new list by removing all null elements from the given list.
+     *
+     * @param      <E> the element type
+     * @param list the input list, must not be null and must not be empty
+     * @return a new list by removing all null elements from the given list
+     * @throws NullPointerException     if the input list is null
+     * @throws IllegalArgumentException if the input list is empty
+     */
+    public static <E> List<E> removeNullElements(final List<E> list) {
+        Objects.requireNonNull(list, "input list must not be null");
+        if (list.size() == 0) {
```

Comment:
This should not be an error IMO. Who cares if nothing was done to the input list after all? 
I still don't see the value, for my money that is.

Reply:
Thanks for the suggestion, fixed the same now.